### PR TITLE
PLANNER-1718: Add runLocally.sh script

### LINKED ISSUE #134: Add buildAndRun.sh script
https://issues.jboss.org/browse/PLANNER-1718.

The goal is to get the application running with a single command to stream-line demoing on video conferences, live meetings etc.

- easy to use
- quick
- development vs. production mode - doesn't matter

### PR DESCRIPTION
**Pull request by yurloc**
Fixes #134.

### CODE REVIEW

File: runLocally.sh [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+  if [[ ! -f ${cc_file} ]]
+  then
+    region=${region%-latest}
+    region=${region//-/ }
+
+    local cc_tag="nv-i18n-1.27"
+    local cc_java="$vrp_dir/CountryCode-$cc_tag.java"
+    [[ ! -f ${cc_java} ]] && \
+      curl https://raw.githubusercontent.com/TakahikoKawasaki/nv-i18n/${cc_tag}/src/main/java/com/neovisionaries/i18n/CountryCode.java -s > "$cc_java"
```
Comment (ge0ffrey) [inline runLocally.sh:93]: What does this do?

**Comment by ge0ffrey** [comment]
Looks good!
I am looking forward to the readme :)

File: runLocally.sh (hunk 1 above)
Comment (yurloc) [reply runLocally.sh]: This downloads a Java enum that is used to guess a country code from the OSM file name.

**Comment by yurloc** [comment]
@michaltomco Feel free to try this out if you have time. My recommendation is:
- See if you can use the script just by following the simple instructions in readme.
- Check out the documentation chapter and see if makes sense and if the script behaves according to the expectations set by the docs.

No need to review the Bash code. It's not nice but should have a reasonable level of quality. I've checked it with [ShellCheck](https://www.shellcheck.net/).

**Comment by yurloc** [comment]
@ge0ffrey documentation done. Also added very simple instructions to the readme. You can give it another try. It's now possible to download any region in the world.

**Comment by yurloc** [comment]
@emmurphy1 please review the documentation chapter if you find the time for it.

**Pull request merged**